docs/kitematic/next_steps.md

```diff
@@ -0,0 +1,34 @@
+<!--[metadata]>
++++
+title = "Where to learn more"
+description = "Explains next steps after the tour"
+keywords = ["Kitematic, open source, contribute, contributor, tour, development, contribute"]
+[menu.main]
+parent= "smn_kitematic_contrib"
+weight=6
++++
+<![end-metadata]-->
+
+# Where to learn more
+
+Congratulations, you've created your first pull request to Kitematic.
+
+The next
+step is for you learn how to [participate in your PR's
+review](http://docs.docker.com/project/review-pr/).
+
+## Take the development challenge
+
+Now that you've had some practice, try adding another feature to Kitematic on your own.
+
+TBD dev feature challenge
```

Comment:
I did on the gist we shared:
https://gist.github.com/FrenchBen/455d5d9497e9522b2668